agent/handlers/v1_handlers_test.go

```diff
@@ -44,172 +44,226 @@ func TestMetadataHandler(t *testing.T) {
 	var resp MetadataResponse
 	json.Unmarshal(w.Body.Bytes(), &resp)
 
-	if resp.Cluster != TestClusterArn {
+	if resp.Cluster != testClusterArn {
 		t.Error("Metadata returned the wrong cluster arn")
 	}
-	if *resp.ContainerInstanceArn != TestContainerInstanceArn {
+	if *resp.ContainerInstanceArn != testContainerInstanceArn {
 		t.Error("Metadata returned the wrong cluster arn")
 	}
 }
 
-func getResponseBodyFromLocalHost(url string, t *testing.T) []byte {
-	resp, err := http.Get("http://localhost:" + strconv.Itoa(config.AGENT_INTROSPECTION_PORT) + url)
-	if err != nil {
-		t.Fatal(err)
+func taskDiffHelper(t *testing.T, expected []*api.Task, actual TasksResponse) {
```

Comment:
Can you move this below the tests that are using it?  I generally prefer that the caller comes before the callee.